examples/github-stars-example/github-stars-example-job/00_properties.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+from vdk.api.job_input import IJobInput
+
+log = logging.getLogger(__name__)
+
+
+def run(job_input: IJobInput):
+    properties = job_input.get_all_properties()
```

Comment:
no need to explicitly load the properties before setting them, in case the loaded (properties var) are not used;
you can drop this line

Reply:
file removed